modules/tidb/src/test/java/org/testcontainers/junit/tidb/SimpleTiDBTest.java

```diff
@@ -0,0 +1,56 @@
+package org.testcontainers.junit.tidb;
+
+import org.junit.Test;
+import org.testcontainers.TiDBTestImages;
+import org.testcontainers.containers.TiDBContainer;
+import org.testcontainers.db.AbstractContainerDatabaseTest;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+import static org.hamcrest.Matchers.containsString;
+import static org.junit.Assert.assertThat;
+import static org.rnorth.visibleassertions.VisibleAssertions.assertEquals;
+
+public class SimpleTiDBTest extends AbstractContainerDatabaseTest {
+
+    @Test
+    public void testSimple() throws SQLException {
+        try (TiDBContainer tidb = new TiDBContainer(TiDBTestImages.TIDB_IMAGE)) {
+            tidb.start();
+
+            ResultSet resultSet = performQuery(tidb, "SELECT 1");
+
+            int resultSetInt = resultSet.getInt(1);
+            assertEquals("A basic SELECT query succeeds", 1, resultSetInt);
+        }
+    }
+
+    @Test
+    public void testExplicitInitScript() throws SQLException {
+        try (
+            TiDBContainer tidb = new TiDBContainer(TiDBTestImages.TIDB_IMAGE).withInitScript("somepath/init_mysql.sql")
```

Comment:
```suggestion
            TiDBContainer tidb = new TiDBContainer(TiDBTestImages.TIDB_IMAGE).withInitScript("somepath/init.sql")
```

Should we rename the file like this? Initially, it felt to me like a copy-paste error, to have the `init_mysql.sql` file used here. Or is this supposed to explicitly communicate MySQL syntax to the maintainers of the module?

Reply:
Yes, I'll change this file to `somepath/init_tidb.sql`